en/docs/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-590-to-is-5110.md

```diff
@@ -214,3 +214,8 @@ Follow the steps mentioned in [Upgrading API-M from 3.0.0 to 4.0.0]({{base_path}
     [user_store]
     type = "database_unique_id"
     ```
+    
+!!! note
+    Before API Manager 4.0.0, the distributed deployment comprised of five main product profiles. They are Publisher, Devportal, Gateway, Key manager, and Traffic Manager. Along with the new architecture in APIM 4.0.0 we only have three profiles such as Gateway, Traffic Manager and Default.
+    Since all the data contains in databases in 4.0.0 executing the migration client  in Default profile is recomamded and it will migrate the data in the connected databases.
```

Comment:
extra space after "client" ?
How about to change like this.
Since API-M 4.0.0 all the data are persisted in databases,  it is recommended to execute the migration client in the Default profile.

Reply:
+1 . I'll do the change

Reply:
503765ed1ef9ecca8b5047ebc2eaad66d0d86943